### DISCUSSION TITLE
using `default write` to restore config

### DISCUSSION THREAD

**Discussion by xbeta** [rxhanson/Rectangle#569] [Q&A]
I would like to know whether there's a way to restore the config file from using macOS command such as `default write` on those shortcut hot-keys? 

I would like to automatically provision macOS setup using bunch of `brew install` and also auto setup this.

**Comment by rxhanson** [comment]
Since all of Rectangle’s config is stored in NSUserDefaults, you can move around the plist that contains them. See https://github.com/rxhanson/Rectangle#preferences-storage
